src/components/OrderColumns.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext, useState } from 'react';
+import PlanetsContext from '../context/Planets';
+
+function OrderColumns() {
+  const { handleOrderColumns } = useContext(PlanetsContext);
+  const initialOrder = { column: 'name', sort: 'ASC' };
+  const [order, setOrder] = useState(initialOrder);
```

Comment:
if you'd need to change the initialOrder, you would need to change in two places

I suggest to use an approach like single source of truth (SSOT) so in this case you'd only need to change the initial values on context, like so:

```suggestion
  const { handleOrderColumns, filters } = useContext(PlanetsContext);
  const [order, setOrder] = useState(filters.order);
```